Tasks/DockerV2/dockerbuild.ts

```diff
@@ -1,19 +1,28 @@
 "use strict";
 
 import * as fs from "fs";
-import * as path from "path";
 import * as tl from "azure-pipelines-task-lib/task";
 import ContainerConnection from "azure-pipelines-tasks-docker-common-v2/containerconnection";
 import * as dockerCommandUtils from "azure-pipelines-tasks-docker-common-v2/dockercommandutils";
 import * as fileUtils from "azure-pipelines-tasks-docker-common-v2/fileutils";
 import * as pipelineUtils from "azure-pipelines-tasks-docker-common-v2/pipelineutils";
 import * as utils from "./utils";
 
+interface ImageAnnotations{
+    BaseImageName :string,
+    BaseImageDigest: string
+}
+
 export function run(connection: ContainerConnection, outputUpdate: (data: string) => any, isBuildAndPushCommand?: boolean): any {
     // find dockerfile path
     let dockerfilepath = tl.getInput("Dockerfile", true);
     let dockerFile = fileUtils.findDockerFile(dockerfilepath);
-    
+    let imageAnnotations: ImageAnnotations = null;
+
+    if(isBaseImageLabelAnnotationEnabled()){
```

Comment:
I'd like to know if there is a way that I can create a test setup to test this scenario.